azure-pipelines.yml

```diff
@@ -61,22 +62,24 @@ jobs:
         feedsToUse: 'config'
         nugetConfigPath: '.nuget/nuget.config'
 
-    - task: VSBuild@1
+    - task: DotNetCoreCLI@2
```

Comment:
Why is this task name change necessary?

Reply:
This version allows you to run arbitrary `dotnet` cli commands. This allows me to build individual projects instead of the whole solution, which saves about 2 minutes per run.